yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -105,6 +92,13 @@ func assessMigrationBulk() {
 
 	for _, dbConfig := range dbConfigs {
 		utils.PrintAndLog("\nAssessing '%s' schema\n", dbConfig.Schema)
+
+		// handling ignore if exists
+		if checkMigrationAssessmentForConfig(dbConfig) {
+			utils.PrintAndLog("assessment report for schema %s already exists, skipping...", dbConfig.Schema)
```

Comment:
Let's use the database identifier here instead of schema, as schema can be ambiguous

Reply:
I think you mean `databaseIdentifier + schema`
Makes sense.